docs/output-api.md

```diff
@@ -108,7 +108,7 @@ for every output handler op.
 
 ## Output handler hooks
 
-The output handler can change its abilities at runtime. Eg. the gz handler can
+The output handler can change its abilities at runtime. e.g. the gz handler can
```

Comment:
```suggestion
The output handler can change its abilities at runtime. E.g. the gz handler can
```

Reply:
Neither please. Use for example

Reply:
I think that makes more sense Kamil. Will update that. Thanks!